javascriptv3/example_code/cognito/scenarios/basic/index.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import { createInterface } from "readline";
+import { stdin as input, stdout as output } from "process";
+import { startsWith } from "ramda";
+
+import { readCommands } from "../../../libs/cmd-runner.js";
+import { log } from "../../../libs/utils/util-log.js";
+import { createUserPoolHandler } from "./command-handlers/create-user-pool-handler.js";
+import { cleanUpHandler } from "./command-handlers/clean-up-handler.js";
+import { signUpHandler } from "./command-handlers/sign-up-handler.js";
+import { confirmSignUpHandler } from "./command-handlers/confirm-sign-up-handler.js";
+import { adminInitiateAuthHandler } from "./command-handlers/admin-initiate-auth-handler.js";
+import { verifySoftwareTokenHandler } from "./command-handlers/verify-software-token-handler.js";
+import { adminRespondToAuthChallengeHandler } from "./command-handlers/admin-respond-to-auth-challenge-handler.js";
+import { adminGetUserHandler } from "./command-handlers/admin-get-user-handler.js";
+import { listUsersHandler } from "./command-handlers/list-users-handler.js";
+import { resendConfirmationCodeHandler } from "./command-handlers/resend-confirmation-code-handler.js";
+
+const cmdInterface = createInterface({ input, output });
+
+const help = `
+ usage: <command> [<args>]
+ 
+   Commands:
+ 
+   help                                   Shows this list of commands.
+
+   create-user-pool <user_pool_name>      Creates and configures a Cognito User Pool
+                                          compatible with this example.
+
+   sign-up <user_name> <password>         Creates a new user in the User Pool.
```

Comment:
Update to description: 
Creates a new user in the user pool.
Also emails a verification code to 
the provided email address.